driver/main.c

```diff
@@ -1229,23 +1230,6 @@ static int ppm_mmap(struct file *filp, struct vm_area_struct *vma)
 		       length,
 		       PAGE_SIZE);
 
-		/*
-		 * Enforce ring buffer size
-		 */
-		if (RING_BUF_SIZE < 2 * PAGE_SIZE) {
```

Comment:
Was this moved elsewhere? I mean, it is now up to userspace to give kmod a correct param, right?

Reply:
I don't see a big issue here since the size can come only from sinsp, and it is well validated at the beginning, but I can restore this check if you think we really need it :thinking:

Reply:
It can also come from `scap`  that's why i'd move the validation in scap :)